### ISSUE TITLE
configuration/setup error : Please define a type for field `credentialsExpired`

### ISSUE THREAD

**Issue by emillosanti** [sonata-project/SonataUserBundle#553]
error message :

Please define a type for field `credentialsExpired` in `Sonata\UserBundle\Admin\Document\UserAdmin` 
- this only happens when installing sonata-admin + sonata-user + fos-user + mongodb
- inspecting Admin/Model/UserAdmin's configureFormFields method :
  
  ```
  if ($this->getSubject() && !$this->getSubject()->hasRole('ROLE_SUPER_ADMIN')) {
      $formMapper
      ...
                  ->add('credentialsExpired', null, array('required' => false))
  ```
  - error exists even when condition is suppressed.

**Comment by emillosanti** [comment]
replacing null with 'text' skipped this error

**Comment by OskarStark** [comment]
this should be a boolean field, right?

try 'boolean' please

**Comment by OskarStark** [comment]
Closed via #869
